tests/integration/test_attachments.py

```diff
@@ -116,6 +116,74 @@ def test_mixed_attachments_with_processing(
     }
 
 
+def test_attachments_with_objectstore(
+    mini_sentry, relay_with_processing, attachments_consumer, outcomes_consumer
+):
+    project_id = 42
+    event_id = "515539018c9b4260a6f999572f1661ee"
+
+    mini_sentry.global_config["options"][
+        "relay.objectstore-attachments.sample-rate"
+    ] = 1.0
+    mini_sentry.add_full_project_config(project_id)
+
+    options = {
+        "processing": {
+            "attachment_chunk_size": "100KB",
+            "upload": {"objectstore_url": "http://127.0.0.1:8888/"},
+        }
+    }
+    relay = relay_with_processing(options)
+    attachments_consumer = attachments_consumer()
+    outcomes_consumer = outcomes_consumer()
+
+    chunked_contents = b"heavens no" * 20_000
+    attachments = [
+        ("att_1", "foo.txt", chunked_contents),
+        ("att_2", "foobar.txt", b""),
+    ]
+    relay.send_attachments(project_id, event_id, attachments)
+
+    attachment = attachments_consumer.get_individual_attachment()
+    assert attachment["attachment"].pop("id")
+
+    # this means the attachment is stored in objectstore
+    assert attachment["attachment"].pop("stored_id")
```

Comment:
Can we make a request to objectstore here to verify that the attachment has actually been stored?